SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -81,7 +83,7 @@ ServerStatusWatcher {
 				onComplete.value;
 			});
 
-		}.play(AppClock)
+		}.play(clock.asClass)
```

Comment:
remove `asClass`.

Reply:
Thank you, and sorry for forgetting. Done! https://github.com/supercollider/supercollider/pull/6198/commits/cb686efe3a28162bf8a55a4aef44c1126967243f